libraries/Parsers/Microsoft.Bot.Builder.Parsers.LU/Parser/LUResource.cs

```diff
@@ -0,0 +1,407 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma warning disable CA2227
+#pragma warning disable CA1716
+#pragma warning disable SA1402
+#pragma warning disable SA1601
+#pragma warning disable SA1602
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Microsoft.Bot.Builder.Parsers.LU.Parser
+{
+    public enum SectionType
+    {
+        SimpleIntentSection,
+        NestedIntentSection,
+        EntitySection,
+        NewEntitySection,
+        ImportSection,
+        ModelInfoSection,
+        QnaSection
+    }
+
+    public enum TypeEnum
```

Comment:
Enums are fragile, and we should generally avoid them. They are not forward compatible, as you cannot add a new enum in the future w/o breaking compat.

Reply:
This was done by an autogen tool, we could discuss a refactor